numeric-keyboard/src/main/java/com/konaire/numerickeyboard/NumericKeyboard.kt

```diff
@@ -222,15 +222,20 @@ class NumericKeyboard : FrameLayout {
         keyRemove.setOnClickListener(listener)
         keyRemove.setOnTouchListener { view, event ->
             when (event.action) {
-                MotionEvent.ACTION_DOWN -> handler.postDelayed(removeChar.also { it.view = view }, 750)
-                MotionEvent.ACTION_UP -> handler.removeCallbacks(removeChar)
+                MotionEvent.ACTION_DOWN -> handler.postDelayed(
+                    removeChar.also { it.view = view },
+                    750
+                )
+                MotionEvent.ACTION_UP, MotionEvent.ACTION_CANCEL ->
+                    handler.removeCallbacks(
+                        removeChar
+                    )
```

Comment:
```suggestion
                    handler.removeCallbacks(removeChar)
```

I think there is no need to move `(removeChar)` to a separate lines

Reply:
Yeah, agreed! I'd commit the suggestion.